templates/injector.html.twig

```diff
@@ -5,14 +5,12 @@
     $R.options = {
         callbacks: {
             image: {
-                uploadError: function(response)
-                {
+                uploadError: function(response) {
```

Comment:
@Thibault-MTZ This is an example of a change that has no impact on the functionality and needs to be adjusted.